brackend/tasks/tasks.py

```diff
@@ -89,11 +65,28 @@ def get_tournament_ids():
         return [u.id for u in users]
 
 
-def get_tournament_by_id(id):
+def get_tournament_by_id(t_id):
     engine = EngineGetter.get_or_create_engine()
     with Session(engine) as session:
-        tournament = session\
-            .query(Tournament)\
-            .filter(Tournament.id == id)\
-            .one_or_none()
+        tournament = session.query(Tournament).filter(Tournament.id == t_id).one_or_none()
```

Comment:
I'd actually recommend keeping queries multi-line if possible. Makes it a lot easier when adding more clauses as well as when reviewing changes made to queries if it's they are split up by one operation per line

Reply:
yeah, `black` (my formatter) does this automatically. I'll revert it back